src/team/teamMembers.js

```diff
@@ -0,0 +1,53 @@
+const TeamMembersObj = {
```

Comment:
To convert to ES2016 from CommonJS, change this line to `export const TeamMembersObj = {` and remove the last line in this file.